lib/InteractiveViewer.mli

```diff
@@ -0,0 +1,21 @@
+(** This module defines the main user interface and interaction logic for a diff viewer application using the Nottui library. It provides mechanisms to navigate through patches and view them in different formats, respond to user inputs, and run test simulations. *)
```

Comment:
"Using the nottui library" is an implementation detail, irrelevant to the user here! ;)

Reply:
Edit the comment